target/linux/ramips/dts/kng_rc.dts

```diff
@@ -0,0 +1,187 @@
+#include <dt-bindings/input/input.h>
```

Comment:
move the includes after ``#include "mt7620a.dtsi"`` and order them alphabetical.